src/applications/facility-locator/containers/FacilitiesMap.jsx

```diff
@@ -70,19 +70,26 @@ const FacilitiesMap = props => {
       return;
     }
     const { location } = props;
+    const {
+      query: { address, context, facilityType, serviceType },
+    } = location;
+    const expandedRadius = facilityType === 'benefits' && !serviceType;
 
     if (!isEmpty(location.query)) {
```

Comment:
If it is, indeed, possible that `location.query` is empty, then line 74 above might also produce an error. Object destructuring can be a very good tool, but it needs some caution.

Reply:
This is a good catch with the destructuring I didn't notice before. I also saw that the `expandedRadius` declaration should be inside the `location.query.address` conditional - it doesn't make sense to create the variable before the code even knows if it's going to be used or not.